go_covert_lib/client/src/ui-components/IPInput.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import FormControl from 'react-bootstrap/FormControl';
+import InputGroup from 'react-bootstrap/InputGroup';
+
+
+const IPInput = (props) => {
+  const { label, value, onChange } = props;
+  return (
+    <InputGroup className="m-1 w-25">
+      <InputGroup.Prepend>
+        <InputGroup.Text className="input-text">{label}</InputGroup.Text>
+      </InputGroup.Prepend>
+      <FormControl
+        value={value}
+        onChange={onChange}
+      />
+    </InputGroup>
+  );
+};
+
+IPInput.propTypes = {
+  label: PropTypes.string.isRequired,
+  value: PropTypes.string,
```

Comment:
You should validate the string in this input to make sure that it is a valid IPv4 address.